app/assets/javascripts/admin/addon/components/admin-user-field-item.hbs

```diff
@@ -1,134 +1,163 @@
-<div class="user-field">
-  {{#if (or this.isEditing (not this.userField.id))}}
-    <AdminFormRow @label="admin.user_fields.type">
-      <ComboBox
-        @content={{this.fieldTypes}}
-        @value={{this.buffered.field_type}}
-        @onChange={{fn (mut this.buffered.field_type)}}
-      />
-    </AdminFormRow>
+{{#if (or this.isEditing (not @userField.id))}}
```

Comment:
To further torment you, the next move here is to move editing user fields to a completely separate route instead of having this massive component with if/else. A lot of these old admin pages are like this, and it's not great since you cannot directly go to a field via URL to edit it. I've had to do this for a bunch of pages/plugins so far.

Reply:
Then you can also have 1 GJS component for the form, and this component can be converted to GJS because it will be "lighter"

Reply:
Created a TODO for this. 👍